src/atoms/icons/iconWrapper.tsx

```diff
@@ -8,13 +8,14 @@ const IconComponent = Styled.div<IconWrapperProps>`
   width: ${(props) => props.width};
   display: flex;
   align-items: center;
+  transition: width 0.3s;
```

Comment:
I added this as experiment, imo when we resize the icons having them grow/shrink seems to look smooth and have that  nice polished look. If you guys notice any weird behavior as side effect on the icons its probably that, so let me know and i can remove it. But hopefully we will like it. :) (I checked the storybooks and based on what i have noticed nothing broke after that change)